libs/hdf-converters/src/jsonix-intermediate-converter.ts

```diff
@@ -0,0 +1,9 @@
+import {JsonixConverter} from './jsonix-converter';
```

Comment:
have a section in the wiki on when to use/not use the intermediate converter

Reply:
Section was added to new page: https://github.com/mitre/heimdall2/wiki/HDF-Converters-How-Tos